scripts/statstab.gd

```diff
@@ -1802,14 +1814,50 @@ func talkconsent(mode=''):
 	if mode == "intro":
 		#Change Dialogue
 		text += "[color=red]Consent Topics are once per day. If Consent failed, you can try again tomorrow.[/color]\n\n[color=yellow]-What did you want to talk about?[/color]"
+		#ralph7
```

Comment:
not sure whether to restrict this one, but I'll note it for now. changed description without condition

Reply:
I'll just gate it behind useRalphsTweaks for now, wouldn't mind revisting this.